src/app/forms/custom-sections/body/body.component.ts

```diff
@@ -47,8 +44,17 @@ export class BodyComponent implements OnInit, OnDestroy {
       }
       this.formChange.emit(event);
     });
-
+    console.log(this.form);
```

Comment:
console log ?

Reply:
I deleted that like... a minute after I created the PR... I swear